flake.nix

```diff
@@ -2,15 +2,15 @@
   inputs = {
     devenv.url = "github:cachix/devenv";
     flake-parts.url = "github:hercules-ci/flake-parts";
-    nixobolus.url = "github:ponkila/nixobolus";
+    homestakeros.url = "github:ponkila/HomestakerOS\?dir=modules/homestakeros";
```

Comment:
off-topic: Why is the question mark escaped? Is this for shell scripts or something?

Reply:
a habit, and the syntax highlighting is nice